lib/custom_widgets/fifth_custom_product.dart

```diff
@@ -0,0 +1,164 @@
+import 'package:flower_app/utils/constant_colors.dart';
+import 'package:flutter/material.dart';
+
+class FifthCustomProduct extends StatefulWidget {
+  final Color color;
+  final String title;
+  final String image;
+  final String image2;
+  final String image3;
+  final String subtitle;
+  const FifthCustomProduct({
+    super.key,
+    required this.title,
+    required this.image,
+    required this.subtitle,
+    required this.image2,
+    required this.color,
+    required this.image3,
+  });
+
+  @override
+  State<FifthCustomProduct> createState() => _PrimaryProductState();
+}
+
+class _PrimaryProductState extends State<FifthCustomProduct> {
+  @override
+  Widget build(BuildContext context) {
+    var width = MediaQuery.of(context).size.width;
+    var height = MediaQuery.of(context).size.height;
```

Comment:
Never use var as a dataType you have to be specific in dataType like bool, double, int ... like this you must know what will be the dataType of variable that you are creating.
use it like this and also change name:
```double deviceWidth = MediaQuery.of(context).size.width;```